### ISSUE TITLE
SET and REPLACE deprecated?

### ISSUE THREAD

**Issue by zaguini** [memcachier/memjs#97]
I get these messages when I run, and it's ok! It's a warning. But I can't find the new way to do the things. It's not in the docs (I'm currently using what is in the docs), and it doesn't seem to be anywhere.

Where can I get info?

**Comment by Shudrum** [comment]
The documentation seems to not be up to date with the README file.

On the doc you can read SET work this way:

```javascript
client.set('testKey', 'testValue', (err, result) => {
    assert.isNull(err);
    done();
}, { expire: 60 });
```

But on the README file:

```javascript
client.set('testKey', 'testValue', { expire: 60 }, (err, result) => {
    assert.isNull(err);
    done();
});
```

The second one do not log any warning. But the mandatory object parameter is a bit annoying.

**Comment by saschat** [comment]
Fixed the README in #112
